src/io/result_io.jl

```diff
@@ -359,9 +362,12 @@ function _recreate_stats_from_store(zarr_store_path::String)::Dict{String,Abstra
         store = zopen(sd, fill_as_missing=false)
 
         dims = store.attrs["structure"]
```

Comment:
Alternatively, you could do `Symbol.(store.attrs["structure"])` and call this `dim_names` then when you call `NamedDimsArray` you just pass `dim_names`. (Another comment you can ignore if you prefer the way it is ;) )